react/features/invite/components/AddPeopleDialog.web.js

```diff
@@ -218,6 +224,49 @@ class AddPeopleDialog extends Component {
                 });
             });
         }
+
+        return true;
+    }
+
+    /**
+     * Renders the error message if the add doesn't succeed.
+     *
+     * @returns {ReactElement|null}
+     * @private
+     */
+    _renderErrorMessage() {
+        if (!this.state.addToCallError) {
+            return null;
```

Comment:
I've actually been debating for a while if these return nulls should happen within the helper or within the actual render's JSX. It kind of makes sense to me that they happen in render so that the helpers always try to execute one code flow. What do you think?